### PR TITLE
add cert-manager to addons list

### PR DESCRIPTION
**Pull request by rasheedamir**
add cert-manager to addons list

### CODE REVIEW

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-204-0a9bf736`

File: content/sre/addons/introduction.md [hunk 1]
```diff
@@ -14,15 +14,16 @@ Code inspection | SonarQube
 Log alerting | [Stakater Konfigurator](https://github.com/stakater/Konfigurator)
 Internal alerting | AlertManager
 External (downtime) alerting | [Stakater IMC](https://github.com/stakater/IngressMonitorController), UptimeRobot (free tier)
-Service mesh | Istio, Kiali, Jaeger (only one control plane)
+Service mesh | Istio, Kiali, Jaeger (only one fully manageed control plane)
```
Comment (TehreemNisa) [inline content/sre/addons/introduction.md]: managed*

**Review by TehreemNisa — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-204-834c4e49`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.147`

**Pull request merged**